keeper-gradle-plugin/src/main/kotlin/com/slack/keeper/KeeperPlugin.kt

```diff
@@ -258,3 +277,29 @@ internal fun String.capitalize(locale: Locale): String {
   }
   return this
 }
+
+private fun unwrapTestedVariant(variantData: Any?): BaseVariant? {
+  return when (variantData) {
+    is BaseVariant -> {
+      when (variantData) {
+        is TestVariant -> variantData.testedVariant
+        is UnitTestVariant -> variantData.testedVariant as? BaseVariant
+        else -> variantData
+      }
+    }
+    is KaptVariantData<*> -> unwrapTestedVariant(variantData.variantData)
```

Comment:
`appExtension.testVariants` only contains `TestVariant`s (which `KaptVariantData` doesn't implement), right? I'm not following why this is necessary -- is this an AGP 4.0.0 thing?

Reply:
Ahh good call. Copy-pasta from the redacted plugin code I borrowed this from

Reply:
0bcc473